standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -2877,6 +2877,20 @@ private static String normalizeCase(String s) {
     return s == null ? null : s.toLowerCase();
   }
 
+  private static String normalizePartitionCase(String s) {
```

Comment:
@nareshpr, LGTM, however could you please try to reuse FileUtils.makePartName(List<String> partCols, List<String> vals)